cli/cli/commands/kurtosis_context/ls/ls.go

```diff
@@ -0,0 +1,98 @@
+package ls
+
+import (
+	"context"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/lowlevel"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/lowlevel/args"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/lowlevel/flags"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_str_consts"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/helpers/output_printers"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/uuid_generator"
+	contexts_config_api "github.com/kurtosis-tech/kurtosis/contexts-config-store/api/golang"
+	contexts_config_generated_api "github.com/kurtosis-tech/kurtosis/contexts-config-store/api/golang/generated"
+	"github.com/kurtosis-tech/kurtosis/contexts-config-store/store"
+	"github.com/kurtosis-tech/stacktrace"
+)
+
+const (
+	contextCurrentColumnHeader = ""
+	contextUuidColumnHeader    = "UUID"
+	contextNameColumnHeader    = "Name"
+	contextRemoteColumnHeader  = "Remote"
```

Comment:
nit: WDYT on renaming to `Remote host`? maybe??

Reply:
Yeah, that makes sense to me. I'll rename! That's something we can easily change anyway :)